### ISSUE TITLE
'ok': False, 'error_code': 400, 'description': 'Bad Request: there is no photo in the request'

### ISSUE THREAD

**Issue by lickceo** [EdNovas/emby_notify_telegram_bot#2]
In 4.8.3, I used this program, but there was an error in the title prompt. I couldn't find the reason. I looked at the image and got it, but there was an error in passing it to TG!

**Comment by EdNovas** [comment]
```
item_img = response['ImageTags']['Primary']
item_img_url = f'{EMBY_SERVER_URL}/emby/Items/{item_id}/Images/Primary?maxHeight=900&maxWidth=600&tag={item_img}
```
      
program uses the photo tagged Primary. If no photo is found, then it means that this video has no photo along with it.

**Comment by lickceo** [comment]
But I have already obtained the photo address using the print function, and there are photos in EMBY's movies and TV series!

**Comment by lickceo** [comment]
But I have already obtained the photo address using the print function, and there are photos in EMBY's movies and TV series!

**Comment by lickceo** [comment]
help me thanks

**Comment by lickceo** [comment]
#上新
Title: 反派大小姐等级99～我是隐藏BOSS但不是魔王～ 
类型: Unknown

频道 https://t.me/*****
群组 https://t.me/***** https://******/emby/Items/859762/Images/Primary?maxHeight=900&maxWidth=600&tag=53a64cd5e7fcf36b153c2aeca81a61ba&keepAnimation=true&quality=90
{'ok': False, 'error_code': 400, 'description': 'Bad Request: there is no photo in the request'}
#上新
Title: 炼气十万年 
类型: Unknown

频道 https://t.me/*****
群组 https://t.me/***** https://******/emby/Items/679242/Images/Primary?maxHeight=900&maxWidth=600&tag=ad54a6474e2117782a23cc462cdcd75c&keepAnimation=true&quality=90
{'ok': False, 'error_code': 400, 'description': 'Bad Request: there is no photo in the request'}
#上新
Title: 星球大战：异等小队（又名：又名: 星球大战：残次品 / 星际大战：瑕疵品 / 星球大战：特种兵团(港) / 星际大战：瑕疵小队(台)） 
类型: Unknown
I have checked and found that the image addresses obtained are all correct, but they will display an error!

**Comment by EdNovas** [comment]
have you tried open the img link directly in browser?

**Comment by lickceo** [comment]
That's right, when I open the link in the browser, there are pictures, and the image address is correct!

**Comment by lickceo** [comment]
![Uploading QQ截图20240319071501.jpg…]()

**Comment by lickceo** [comment]
The images for each address are correct and can be opened in the browser!

**Comment by EdNovas** [comment]
maybe because of firewall blocks, not sure

**Comment by lickceo** [comment]
Is it the firewall of this machine? Do you need to open any ports, or is it?

**Comment by lickceo** [comment]
系统：Ubuntu 20

**Comment by lickceo** [comment]
Can you help me? The server version is 4.8.3, and I really like this program. It's simple and practical, but if I can't use it, I feel frustrated!
